server/lib/matchmaking/models.ts

```diff
@@ -477,7 +477,7 @@ export async function getMatchmakingFinalizedRanksForUser(
       WHERE
         mfr.user_id = ${userId}
       ORDER BY
-        ms.start_date DESC
+        ms.start_date DESC, mfr.matchmaking_type
```

Comment:
We may need to re-sort the enum (if we can?), I'm not sure how it is ordered currently but we weren't ordering it based on an expectation of it sorting the UIs and such as well

Reply:
PostgreSQL [documentation](https://www.postgresql.org/docs/current/datatype-enum.html#DATATYPE-ENUM-ORDERING) says this about the ordering of the enum type:
```
The ordering of the values in an enum type is the order in which the values were listed when the type was
created. All standard comparison operators and related aggregate functions are supported for enums.
```

In our case that makes it `1v1, 2v2, 1v1fastest` so far. Which seems fine to me? Although might get a bit tricky to think about that once we add more matchmaking types.

Also, maybe we should sort by games count in each matchmaking type?